### ISSUE TITLE
Deprecated readme - cannot launch, incomplete docker-compose

### ISSUE THREAD

**Issue by marrrcin** [Hydrospheredata/hydro-serving#67]
Could you please update readme.md, because it provides inconsistent information about how to launch ML Lambda. Your `docker-compose` only contains postgres+manager whereas standalone commands start a few other services too. `Docker-compose` should contain everything, so new users could get up and running quickly. 

Also, you should probably use docker network instead of exposing everything via host network (you put `HOST_IP` environment variable in `docker-compose.yml` file - why?)

One last thing is that you could just push everything to Docker HUB, so new users will not be forced to build everything on their machines.

**Comment by Zajs** [comment]
Hello. 
Thank you for your feedback.
I changed docker-compose file and will merge it to the master branch as soon as I finish readme.md and separate development.md.

We used this docker-compose file + HOST_IP variable for debugging purposes. It is more convenient to debug mesh when you have direct access to each point of this mesh. I will move this docker-compose file to separate "development" section.

We pushed released versions to Docker HUB. At this moment we don't have the latest image version built from the master branch. We will add a new stage to our CI pipeline, which will push this image to Docker Hub.
